src/libraries/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -13,6 +14,29 @@ public static class GuidTests
         private static readonly Guid s_testGuid = new Guid("a8a110d5-fc49-43c5-bf46-802db8f843ff");
         private static readonly Guid s_fullGuid = new Guid(uint.MaxValue, ushort.MaxValue, ushort.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue, byte.MaxValue);
 
+        [Fact]
+        public static unsafe void PerfTest()
+        {
+            Stopwatch sw = new Stopwatch();
+            Guid g = new Guid();
+            for (;;)
```

Comment:
Was this intentional? (I realize the PR is "no review", but I had to look... and I'm not sure how much log output you'll get if the whole thing times out)

Reply:
I believe that the log is still collected, even when things timeout.

Reply:
Okey dokey